lib/api_repo.js

```diff
@@ -226,6 +227,7 @@ ApiRepo.prototype.buildPackages =
                 cleanName.replace(new RegExp('-', 'g'), '/');
             opts.packageInfo.api.name = that.pkgPrefix + cleanName;
             opts.packageInfo.api.version = version;
+            opts.packageInfo.api.buildCommonProtos = buildCommonProtos;
```

Comment:
You can just do `opts.packageInfo.api.buildCommonProtos = that.buildCommonProtos;` and then you don't need line 204.